go_test/parse_test.go

```diff
@@ -45,8 +46,9 @@ func TestParseMetadata(t *testing.T) {
 			assert.Greater(t, len(metadata.Links), 0)
 			linkMap := make(map[string]string)
 			for _, link := range metadata.Links {
+				_, err := url.ParseRequestURI(link.URL)
+				assert.Nil(t, err)
```

Comment:
Do you have an example of what the error message for this will look like? It's important that `link.URL` is included so that one can find and fix the offending URL.

Reply:
Yep, here is what it looks like:

--- FAIL: TestParseMetadata (0.33s)
    --- FAIL: TestParseMetadata/../dom/events/META.yml (0.00s)
        parse_test.go:51: 
            	Error Trace:	parse_test.go:51
            	Error:      	Expected nil, but got: &url.Error{Op:"parse", URL:"bugs.chromium.org/p/chromium/issues/detail?id=826702", Err:(*errors.errorString)(0xc0002449c0)}
            	Test:       	TestParseMetadata/../dom/events/META.yml

Reply:
Great, thanks!